demisto_sdk/scripts/validate_content_path.py

```diff
@@ -283,6 +290,9 @@ def _validate(path: Path) -> None:
         ):
             raise InvalidXSIAMReportFileName
 
+        if first_level_folder == DOC_FILES_DIR:
```

Comment:
```suggestion
        if first_level_folder == DOC_FILES_DIR and path.suffix in (".png", ".jpg", ".jpeg", ".gif"):
```
1. Have I missed any suffixes?
2. Looks like we call it on all files under `DOC_FILES_DIR`. Please insert this change and then modify the unit tests to match (make sure that it only fails on image files, test it with image suffixes and with non-image and make sure non-image files can contain chars that images are not allowed to).

Reply:
The constant ALLOWED_SUFFIXES support only jpg and svg